draft-ietf-webtrans-http2.md

```diff
@@ -306,12 +306,14 @@ used to provide protection against traffic analysis or for other reasons.
 WT_PADDING Frame {
   Type (i) = 0x00,
   Length (i),
+  Padding (..),
 }
 ~~~
 {: #fig-wt_padding title="WT_PADDING Frame Format"}
 
-The padding extends to the end of the WT_PADDING frame and that many bytes can
-be discarded by the recipient.
+The Padding field MUST be set to an all-zero sequence of bytes of any length as
+specified by the Length field.
+<!-- TODO validation and error handling -->
```

Comment:
Yeah, I stripped these for basically every frame that had any requirement just until we have some text describing the mechanism for it, then we can drop it back in.